src/prof-http/src/lib.rs

```diff
@@ -87,14 +87,14 @@ pub struct FlamegraphTemplate<'a> {
 }
 
 #[allow(dropping_copy_types)]
-async fn time_prof(
+async fn time_prof<'a>(
```

Comment:
Is the `'a` lifetime parameter needed here? It's only used for the return type. All the arguments are static, so the return type must be static too, no?